src/api-reference/user-provisioning/spend/spend-role-code-definition.md

```diff
@@ -124,7 +123,7 @@ The role code value that can be used for provisioning with the `Spend Role Exten
 |`INV_EMPLOYEE_ADMIN`|Employee Admin Permission on Invoice Hierarchy|The user assigned this role can grant Invoice Hierarchy Nodes permission to users in the Employee Admin.<br>**NOTE**: Each user assigned employee admin must also be assigned permissions for reporting (BI), expense, and payment (Invoice). These additional permissions govern the roles the employee administrator can assign for the employees in his or her assigned groups.|Y|N|
 |`INV_AP_USER`|Invoice AP User|The user assigned this role can create, assign, and reassign invoices. They can also reassign a different policy to an invoice and restore deleted invoices.|Y|N|
 |`INV_APPROVER`|Invoice Approver|The user assigned this role can to approve invoices within an assigned group.|N|N|
-|`INV_CONFIG_ADMIN`|Invoice(Payment) Configuration Administrator|**This role is intended to be assigned to and used by SAP Concur internal staff only.**|Y|N|
+|`INV_CONFIG_ADMIN`|Invoice(Payment) Configuration Administrator|**__This role is intended to be assigned to and used by SAP Concur internal staff only, with few exceptions.__**<br>The user assigned this role can **__fully manage__** (add, edit, delete):<br><ul><li>Invoice group configurations</li><li>Policies</li><li>Invoice-based forms, fields, and validations</li><li>Invoice and authorized approver workflows</li><li>Audit rules</li><li>Expense types</li><li>Account codes and account code hierarchy</li><li>Exceptions</li><li>Image handling, including scan configurations, invoice imaging, vendor imaging</li><li>Email reminders</li><li>Company Info</li><li>Configuration change log (view only)</li></ul>**NOTE:** A user cannot be assigned the admin role and the restricted admin role.|Y|N|
```

Comment:
Does it make sense to also use `<li>` for `INV_CONFIG_ADMIN_RO` on their capabilities as well?

Reply:
This is a very good question, we added those format originally, but it seems like they all got rid of.

Reply:
Thanks, that makes more sense.

Reply:
Heads up, we cannot use the <li> formatting. It will break the tables when it generate to the site, that was why it was removed.